api/src/Page/Exp.php

```diff
@@ -20,7 +20,7 @@ class Exp extends Page
 
             // DiffractionPlan (=>DataCollectionPlan)
             'DIFFRACTIONPLANID' => '\d+',
-            'BLSAMPLEID' => '\d+',
+            // 'BLSAMPLEID' => '\d+',
```

Comment:
There were various args duplicated in the classes - sometimes their definitions changed across the duplication, or it looked like there may be an issue - i.e. there might be a need for two variables to separate out the use between different sets of data.  In the cases I was uncertain, I just commented out the arg rather than deleting - to give a chance for a proper review?